src/widgets-service.js

```diff
@@ -1,4 +1,5 @@
 var _ = require('underscore');
+var URI = require('urijs');
```

Comment:
I agree, but there are a couple more uses than just getting the query parameter, see `src/api/dashboard`